internal/spokes/spokes.go

```diff
@@ -629,11 +676,11 @@ func (r *SpokesReceivePack) report(_ context.Context, unpackOK bool, commands []
 	}
 	for _, c := range commands {
 		if c.err != "" {
-			if err := writePacketf(&buf, "ng %s %s\n", c.refname, c.err); err != nil {
+			if err := writePacketf(&buf, "%s %s %s\n", c.reportFF, c.refname, c.err); err != nil {
```

Comment:
In this case we should always report `ng` right?  It might be easier to hardcode that rather than having to figure out what value the `reportFF` flag is?

Reply:
Sounds fine; this should do it ee6b1dcfb2173bc480d0e49114716499bb1d8058